Extension/ThirdPartyNotices.txt

```diff
@@ -41,6 +41,7 @@ Microsoft C/C++ Extension for Visual Studio Code incorporates components from th
 33.	vscode-extension-telemetry (https://github.com/Microsoft/vscode-extension-telemetry)
 34.	vscode-languageserver-node (https://github.com/Microsoft/vscode-languageserver-node)
 35.	yauzl (https://github.com/thejoshwolfe/yauzl)
+36.	C++11 Sublime Text Snippets (https://github.com/Rapptz/cpp-sublime-snippet)
```

Comment:
Are these supposed to be alphabetized?  I was thinking that our LCA contact usually provided this file.

Reply:
they usually do. @greazer did you just add this or get the entire file from LCA?

Reply:
1) I didn't notice that they were alphabetized. 
2) I just added this line based on how the others were handled.